src/main/java/org/jenkinsci/plugins/scriptler/util/ScriptHelper.java

```diff
@@ -79,6 +79,53 @@ public static Script getScript(String id, boolean withSrc) {
         }
         return s;
     }
+    
+    /**
+     * @since TODO
+     */
+    public static void putScriptInApprovalQueueIfRequired(String scriptSourceCode){
+        // we cannot use sandbox since the script is potentially sent to slaves
+        // and the sandbox mode is not meant to be used with remoting
+        ScriptApproval.get().configuring(scriptSourceCode, GroovyLanguage.get(), ApprovalContext.create().withCurrentUser());
+    }
+    
+    /**
+     * @param scriptSourceCode Source code that must be approved
+     * @return true iff the script was approved or created by a user with RUN_SCRIPT permission
+     * @since TODO
+     */
+    public static boolean isApproved(String scriptSourceCode){
+        return isApproved(scriptSourceCode, true);
+    }
+    
+    /**
+     * @param scriptSourceCode Source code that must be approved
+     * @param putInApprovalQueueIfNotApprovedYet true means we try to know if the user has permission
+     *                                          to approve the script automatically in case it was not approved yet
+     * @return true iff the script is approved
```

Comment:
typo: `iff` -> `if`

Reply:
`iff` = "if and only if", but I will change it as it's not that clear 👍